src/More/CallbackForm.tsx

```diff
@@ -0,0 +1,237 @@
+      Alert.alert(t("common.something_went_wrong"), e.message)
+      setIsLoading(false)
+    }
+  }
+
+  const showError = (error: string): string => {
+    switch (error) {
+      default: {
+        return t("common.something_went_wrong")
+      }
+    }
+  }
+
+  return (
+    <ScrollView
+      style={style.container}
+      contentContainerStyle={style.contentContainer}
+    >
+      <KeyboardAvoidingView
+        keyboardVerticalOffset={Spacing.tiny}
+        behavior={isIOS ? "padding" : undefined}
+      >
+        <View>
+          <View style={style.headerContainer}>
+            <GlobalText style={style.header}>
+              {t("callback_form.well_get_in_touch")}
+            </GlobalText>
+            <GlobalText style={style.subheader}>
+              {t("callback_form.fill_out_the_info", {
+                healthAuthorityName,
+              })}
+            </GlobalText>
+          </View>
+
+          <View style={style.inputContainer}>
+            <GlobalText style={style.inputLabel}>
+              {t("callback_form.firstname")}
+            </GlobalText>
+            <TextInput
+              value={firstname}
+              style={style.textInput}
+              keyboardType={"default"}
+              returnKeyType={"done"}
+              onChangeText={handleOnChangeFirstname}
+              blurOnSubmit={false}
+              onSubmitEditing={Keyboard.dismiss}
+              autoCapitalize={"none"}
+            />
+          </View>
+
+          <View style={style.inputContainer}>
+            <GlobalText style={style.inputLabel}>
+              {t("callback_form.lastname")}
+            </GlobalText>
+            <TextInput
+              value={lastname}
+              style={style.textInput}
+              keyboardType={"default"}
+              returnKeyType={"done"}
+              onChangeText={handleOnChangeLastname}
+              blurOnSubmit={false}
+              onSubmitEditing={Keyboard.dismiss}
+              autoCapitalize={"none"}
+            />
+          </View>
+
+          <View style={style.inputContainer}>
+            <GlobalText style={style.inputLabel}>
+              {t("callback_form.phone_number_required")}
+            </GlobalText>
+            <TextInput
+              value={phoneNumber}
+              style={style.textInput}
+              keyboardType={"phone-pad"}
+              returnKeyType={"done"}
+              onChangeText={handleOnChangePhoneNumber}
+              blurOnSubmit={false}
+              onSubmitEditing={Keyboard.dismiss}
+              multiline
+            />
+          </View>
+
+          <GlobalText style={style.errorSubtitle}>{errorMessage}</GlobalText>
+        </View>
+        {isLoading ? <LoadingIndicator /> : null}
+
+        <Button
+          onPress={handleOnPressSubmit}
+          label={t("common.submit")}
+          loading={isLoading}
+        />
+      </KeyboardAvoidingView>
+    </ScrollView>
+  )
+}
+
+const LoadingIndicator = () => {
```

Comment:
@devinjameson noticing that we have 3 or so of these now, what do you think about introducing a component for them to to be shared. seems _pretty_ self contained to me and not likely to want to diverge based on call site?